collect_app/src/main/java/org/odk/collect/android/utilities/ViewUtil.java

```diff
@@ -0,0 +1,35 @@
+package org.odk.collect.android.utilities;
+
+
+import android.os.Build;
+import android.view.View;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class ViewUtil {
```

Comment:
How about ViewIdGenerator as a name? Isn’t that more descriptive?

Reply:
Was originally trying to keep it in line with our other "Util" classes but I something more descriptive is better. Since it has just the one static method, maybe "ViewIds", so all of the call sites are "ViewIds.generateViewId()"?

Reply:
My own view is that “Util” classes often encourage disorganization. `ViewIds` sounds good!